dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -16,12 +16,10 @@ IUSE=""
 DEPEND=""
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	# This patch was gained from the elfsign-0.2.2 release
-	epatch "${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
-	# Cleanup stuff
-	epatch "${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
-}
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
+	"${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+)
 
 src_compile() {
 	append-flags -fPIC
```

Comment:
do the `append-flags` in `src_configure()` and remove the `|| die` from `emake`

Reply:
Done